src/components/action-chip/package.json

```diff
@@ -7,6 +7,6 @@
   "dependencies": {
     "@bcrs-shared-components/enums": "^1.0.22",
     "@bcrs-shared-components/interfaces": "^1.0.42",
-    "vue-property-decorator": "^8.4.0"
+    "vue-property-decorator": "^9.1.2"
```

Comment:
I figured `lerna bootstrap --hoist` would handle this but I had to do it manually this time 🤷‍♂️